README.md

```diff
@@ -2,16 +2,17 @@
 
 geOrchestra Maelstro is an application which helps synchronise geonetwork and geoserver instances
 
-## Quick start with geOrchestra
+## Docker deploy
+### Quick start with geOrchestra
```

Comment:
Finally I see two options:

* Rename `docker-compose.yml` to `docker-compose-solo.yml`
  - and create `docker-compose.yml` as a symlink before start
  - this is more clear IMHO

* Use an `.env` file with `COMPOSE_FILE=docker-compose-dev.yml:docker-compose.override.yml`
  - `docker-compose.override.yml` is mandatory in this case
  - `cp docker-compose.override.sample.yml docker-compose.override.yml`